config/tailwind.config.js

```diff
@@ -18,13 +23,21 @@ module.exports = {
       colors: {
         black: '#242629',
         offwhite: '#F9FAFB',
+        'account-depository': '#2E90FA',
+        'account-investment': '#32D583',
+        'account-property': '#F23E94',
+        'account-vehicle': '#6172F3',
+        'account-credit': '#36BFFA',
+        'account-loan': '#F38744',
+        'account-other-asset': '#12B76A',
+        'account-other-liability': '#F04438',
```

Comment:
I think we will want to avoid any arbitrary names like this.  Ideally, we'll just have a bunch of standardized color names here in the config.  I think @Shpigford may have a Figma file around this shortly!

Reply:
@Shpigford let me know how we can proceed, especially with the color naming!

Reply:
@elithecho I'm going to push a change to the Tailwind config here shortly that you can use.

Reply:
Color palette is now live - https://github.com/maybe-finance/maybe/pull/452

I've updated these lines so once you pull down `main` you can accept the incoming changes.